common.mk

```diff
@@ -93,14 +93,13 @@ test-openshift: tag
 tag: build
 	VERSIONS="$(VERSIONS)" $(script_env) $(tag)
 
-.PHONY: clean
-clean: clean-images $(clean-hook)
+.PHONY: clean clean-images clean-versions
+clean: clean-images
+	$(MAKE) clean-hook
```

Comment:
Is the sub-make going to inherit all of the defined variables by default?

Reply:
Yes.  I'll rebase once more to add --no-print-directory